resources/odin-doc.json

```diff
@@ -1,5 +1,19 @@
 {
     "collections": {
+        "base": {
+            "name": "base",
+            "source_url": "https://github.com/odin-lang/Odin/tree/master/base",
+            "base_url": "/base",
+            "root_path": "$ODIN_ROOTbase",
```

Comment:
This doesn't look right to me, is that correct?

Reply:
It is yeah, same thing with the other collections below, `$ODIN_ROOT` is search and replaces with the path with a slash suffix.